autobuild/build.sh

```diff
@@ -107,12 +103,21 @@ fi
 if [[ `uname` == "Darwin" ]]; then
 	# autopoint is not in PATH after install via brew (conflicting with system gettext https://github.com/Homebrew/legacy-homebrew/issues/24070)
 	# so we can do `brew link --force gettext` or just add it to PATH before configuring which is preferable because we need it only for compiling
-	export PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/gettext/bin:${PATH}"
-	export LDFLAGS="-L/usr/local/opt/gettext/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libomp)/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libtool)/lib ${LDFLAGS}"
-	export CPPFLAGS="-I/usr/local/opt/gettext/include ${CPPFLAGS}"
-	export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig:${PKG_CONFIG_PATH}"
+	export PATH="$(brew --prefix ccache)/libexec:$(brew --prefix gettext)/bin:${PATH}"
+	# `include` is required to find library headers. For example, currently we are using #include <harfbuzz/hb.h> instead of <hb.h>
+	# but pkg-config returns `harfbuzz/include/harfbuzz`, not just `harfbuzz/include`. This works fine on Linux because
+	# all system libraries is already included by compiler. This need to be fixed. (TODO)
```

Comment:
ugh. Harfbuzz pollutes the include path. Argh. Should we 'fix' our `#include` first?

Reply:
Yes, we need to fix `#include`'s first.

Reply:
#2442

Reply:
Is this comment up to date after merging the PR mentioned in previous comment?

Reply:
Is it still needed after 2442?

Reply:
Thank you for reminding. Fixed.